src/routes/docs/install/+page.md

```diff
@@ -56,6 +56,25 @@ The [Scoop](https://scoop.sh) package is maintained by a third party.
 scoop install nonportable/files-np
 ```
 
+### Offline installers
+
+<InfoBar title="Note" severity="information">
+	The files will need to be installed in the same order as below
+</InfoBar>
+
+```
+https://cdn.files.community/files/stable/Files.Package_4.0.0.0_Test/Dependencies/x64/Microsoft.VCLibs.x64.14.00.Desktop.appx
```

Comment:
This is the `x64` version, should we also include links for `arm64`?

Reply:
I added the arm links, don't know the formatting to make them collapsible

Reply:
Can you confirm it looks good to you